src/protocols/secure_channel/CASESession.h

```diff
@@ -210,6 +213,8 @@ class DLL_EXPORT CASESession : public Messaging::UnsolicitedMessageHandler,
     CHIP_ERROR ValidateSigmaResumeMIC(const ByteSpan & resumeMIC, const ByteSpan & initiatorRandom, const ByteSpan & resumptionID,
                                       const ByteSpan & skInfo, const ByteSpan & nonce);
 
+    bool SanityCheck() const;
```

Comment:
It would be helpful if method had a documentation header.  A more descriptive name would be nice too.